rotkehlchen/chain/evm/decoding/hop/decoder.py

```diff
@@ -60,13 +64,18 @@ def __init__(
             msg_aggregator=msg_aggregator,
         )
         self.bridges = bridges
+        self.swaps = {
+            value.saddle_swap: value.identifier
+            for _, value in bridges.items()
+            if value.saddle_swap is not None
+        }
 
-    def _get_bridgeasset_amount(self, amount_raw: int, bridge: HopBridgeEventData) -> FVal:
+    def _get_bridgeasset_amount(self, amount_raw: int, identifier: str) -> FVal:
```

Comment:
Renamed it so that we can use it in other functions where we just have the identifier, and as the function only needed the asset identifier, renamed it to such. It's basically `bridge_asset_identifier`. Will update.